src/test/java/org/ibp/api/java/impl/middleware/inventory/planting/validator/PlantingRequestDtoValidatorTest.java

```diff
@@ -0,0 +1,255 @@
+package org.ibp.api.java.impl.middleware.inventory.planting.validator;
+
+import org.apache.commons.lang.math.RandomUtils;
+import org.generationcp.middleware.domain.inventory.common.SearchCompositeDto;
+import org.generationcp.middleware.domain.inventory.planting.PlantingRequestDto;
+import org.generationcp.middleware.service.impl.inventory.PlantingPreparationDTO;
+import org.ibp.api.domain.ontology.VariableDetails;
+import org.ibp.api.domain.ontology.VariableFilter;
+import org.ibp.api.exception.ApiRequestValidationException;
+import org.ibp.api.java.impl.middleware.inventory.common.validator.InventoryCommonValidator;
+import org.ibp.api.java.ontology.VariableService;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.springframework.validation.BindingResult;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.hamcrest.CoreMatchers.hasItem;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+@RunWith(MockitoJUnitRunner.class)
+public class PlantingRequestDtoValidatorTest {
+
+	@Mock
+	private org.generationcp.middleware.service.api.inventory.PlantingService plantingService;
+
+	@Mock
+	private InventoryCommonValidator inventoryCommonValidator;
+
+	@Mock
+	private VariableService variableService;
+
+	@InjectMocks
+	private PlantingRequestDtoValidator plantingRequestDtoValidator;
+
+	private static final Integer STUDY_ID = 1;
+
+	private static final Integer DATASET_ID = 1;
+
+	@Before
+	public void setup() {
+		Mockito.doNothing().when(inventoryCommonValidator)
+			.validateSearchCompositeDto(Mockito.any(SearchCompositeDto.class), Mockito.any(BindingResult.class));
+		Mockito.doNothing().when(inventoryCommonValidator).validateUnitNames(Mockito.anyList(), Mockito.any(BindingResult.class));
```

Comment:
I checked and I think you can remove these, the default of any mock object is to _do nothing_